pkg/services/ngalert/store/alert_rule.go

```diff
@@ -275,9 +275,26 @@ func (st DBstore) ListAlertRules(ctx context.Context, query *ngmodels.ListAlertR
 		q = q.Asc("namespace_uid", "rule_group", "rule_group_idx", "id")
 
 		alertRules := make([]*ngmodels.AlertRule, 0)
-		if err := q.Find(&alertRules); err != nil {
+		var faultyRules int64 = 0
+		rule := new(ngmodels.AlertRule)
+		rows, err := q.Rows(rule)
+		if err != nil {
 			return err
 		}
+		defer rows.Close()
```

Comment:
The linter noticed this too, but we can handle the return of close via this pattern instead:
```
defer func() {
    _ = rows.Close()
}()
```
If we really want, we can optionally log the error returned by Close